common/src/stack/report-system/command/report/system/tests/test_sles_patch.py

```diff
@@ -0,0 +1,114 @@
+import pytest
+import os
+import stack.api
+import hashlib
+import json
+
+
+# Returns a hash of filename
+def Get_A_Hash(filename):
+	hasher = hashlib.md5()
+	with open(filename, 'rb') as f:
+		buf = f.read()
+		hasher.update(buf)
+
+	return hasher.hexdigest()
+
+# Compare 2 hashes and return boolean
+def Hashes_Match(hash1, hash2, filename):
+	if hash1 == hash2:
+		return True 
+	else:
+		print('%s hash did NOT match' % filename)
+		return False
+
+# Grab all found SLES versions and returns a list
+def Get_SLES_Flavors():
```

Comment:
I pulled this section out of main body into a function for thinking I will have multiple tests in this file. I can put it back if we are only going to end up with 1 test.